README.md

```
@@ -1,19 +1,95 @@
 
 ```bash
+npm install
 npm run dev
 ```
 
-Or, you could use `docker` to start by
+### Environment Variables
+
+Create a `.env.local` file with:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)